undercloud.yml

```diff
@@ -52,3 +95,17 @@
         args:
             chdir: "{{ infrared_dir }}"
         changed_when: false
+        when: tls_everywhere != true
+
+      - name: run tripleo-undercloud tls
+        shell: |
+          source {{ infrared_dir }}/.venv/bin/activate
+          infrared tripleo-undercloud -vv \
+          --version {{ osp_release }} \
+          --build {{ osp_puddle }} \
+          --images-task rpm \
+          --config-file {{ undercloud_conf }} {{ container_params }} --tls-everywhere true > {{ log_directory }}/undercloud_install.log 2>&1
+        args:
+            chdir: "{{ infrared_dir }}"
+        changed_when: false
+        when: tls_everywhere == true
```

Comment:
AFIK --tls-everywhere {{ tls_everywhere}} in the previous task will work

Reply:
Done